### PR TITLE
Run Wildfly tests only on Java 11

### PR DESCRIPTION
**Pull request by arteam**
The Wildfly docker image supports only Java 11, so we can't use it if the code compiled against a later java version https://github.com/jboss-dockerfiles/wildfly#image-internals-updated-dec-13-2018

Wildfly 25.0.1 does support Java 17, but there's no Java17 docker image for it yet: https://github.com/jboss-dockerfiles/wildfly/issues/148

### CODE REVIEW

**Comment by arteam** [comment]
@elasticmachine run elasticsearch-ci/part-2

**Comment by arteam** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Comment by arteam** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by arteam** [comment]
Also tested that the tests actually get skipped in https://github.com/elastic/elasticsearch/pull/81080

**Comment by arteam** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Review by pugnascotia — APPROVED** [review]
LGTM.

File: qa/wildfly/build.gradle [hunk 1]
```diff
@@ -70,3 +73,9 @@ tasks.register("integTest", Test) {
 tasks.named("check").configure {
   dependsOn "integTest"
 }
+
+static def isDockerImageAvailable() {
+  // The wildfly docker image is only available for x86 architectures so disable these tests on arm
+  // The wildfly docker image is only available for Java 11
```
[the remark this answers is not in the document]
Comment (arteam) [reply qa/wildfly/build.gradle:79]: Sure, I will add a link to https://github.com/jboss-dockerfiles/wildfly/issues/148

**Comment by mark-vieira** [comment]
GIven what this _actually_ testing is the HLRC, which we are no longer publishing in 8.0+ I think we should actually remove these tests altogether.

**Comment by arteam** [comment]
That's a very good point! Let me create a PR removing the Wildfly dependency.

**Comment by mark-vieira** [comment]
I'm going to close this and just remove it as part of https://github.com/elastic/elasticsearch/pull/80851.

**Pull request closed without merging**